templates/form.html

```diff
@@ -52,6 +53,27 @@ <h1 class = "title" >Username Availability Checker</h1>
                 </center>
             </div>
         </div>
+        <div id="usernames">
+            <p class="textualbk"> Need a username? </p>
+            <form id="usernames-form">
+                <label for="min_length">Min Length</label>
+                <input type="number" min="4" max="14" id="min_length" value="6" required> <br/>
+
+                <label for="max_length">Max Length</label>
+                <input type="number" min="4" max="40" id="max_length" value="12" required> <br/>
+
+                <label for="use_underscores">Underscores?</label>
+                <input type="checkbox" id="use_underscores"> <br/>
+
+                <label for="amount">How many usernames?</label>
+                <input type="number" id="amount" min="1" max="20" value="1" required> <br/>
+
+                <input type="submit" value="Generate usernames!">
```

Comment:
Looks like the other form uses a `<button>`, so probably best to keep that consistent.

```suggestion
                <button type="submit">Generate usernames!</button>
```